Makefile

```diff
@@ -0,0 +1,12 @@
+all: docker-compose
+
+generateABI:
+	@solc --abi contracts/erc20.sol -o build --overwrite
+
+transpileABIToGo:
+	@abigen --abi=build/ERC20.abi --pkg=token --out=token/erc20.go
```

Comment:
I would add all generated files to `.gitignore`

Reply:
True. Adding

Reply:
Actually, the `build` folder, where the ABI is generated, is already added. I think we can keep the token in VC so we track changes made to the erc20 contract and their impact. Even though it's generated.

Reply:
Why would we change the ERC20 contract?

Reply:
In case we add more common interface methods into it

Reply:
Ok, we should make a new PR to add an `/interfaces` folder and store Solidity interfaces there. Then we can get rid of generated files in the repo.